staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -173,21 +174,37 @@ func NewDefaultPluginHandler(validPrefixes []string) *DefaultPluginHandler {
 func (h *DefaultPluginHandler) Lookup(filename string) (string, bool) {
 	for _, prefix := range h.ValidPrefixes {
 		path, err := exec.LookPath(fmt.Sprintf("%s-%s", prefix, filename))
-		if err != nil || len(path) == 0 {
+		if shouldSkipOnLookPathErr(err) || len(path) == 0 {
 			continue
 		}
 		return path, true
 	}
-
 	return "", false
 }
 
+func Command(name string, arg ...string) *exec.Cmd {
```

Comment:
godoc comment?

Reply:
since this is a cherry-pick of a commit to master (https://github.com/kubernetes/kubernetes/pull/111254/commits/1f557035e053e2f604b228815241d59123f9915c), I probably wouldn't add that here, but a fix to the file in master is welcome